packages/ecs-gpu-init/ecs-gpu-init-tmpfiles.conf

```diff
@@ -0,0 +1 @@
+d /var/lib/ecs/gpu 0700 root root
```

Comment:
In general, for anything that touches persistent storage, I prefer the combination of:
```
R /var/lib/ecs/gpu - - - -
C /var/lib/ecs/gpu - - - -
Z /var/lib/ecs/gpu 0700 root root -
```

In other words:
* remove the directory if we don't strictly need the persistent state
* create the directory
* relabel the directory according to the active SELinux policy rules

This is somewhat more future-proof since it means that when we downgrade into an older version, we know it won't be confused by newer file formats or unexpected SELinux labels.